doc/source/whatsnew/v0.17.0.txt

```diff
@@ -150,7 +150,7 @@ The Series and DataFrame ``.plot()`` method allows for customizing :ref:`plot ty
 To alleviate this issue, we have added a new, optional plotting interface, which exposes each kind of plot as a method of the ``.plot`` attribute. Instead of writing ``series.plot(kind=<kind>, ...)``, you can now also use ``series.plot.<kind>(...)``:
 
 .. ipython::
-  :verbatim:
+    :verbatim:
```

Comment:
yes, you can choose whatever indentation you want, as long as it is the same for the code below (most of the time, I find using 4 spaces the easiest as then you can use tab)